.github/CODEOWNERS

```diff
@@ -28,6 +28,7 @@
 /codelabs/**/*                         @GoogleCloudPlatform/python-samples-reviewers
 /composer/**/*                         @leahecole @rachael-ds @rafalbiegacz @GoogleCloudPlatform/python-samples-reviewers
 /compute/**/*                          @m-strzelczyk @GoogleCloudPlatform/python-samples-reviewers
+/container/**/*                        @GoogleCloudPlatform/dee-platform-ops @GoogleCloudPlatform/python-samples-reviewers
```

Comment:
Could you also update https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml?